### ISSUE TITLE
Buttons or dropbox to select **Content type** in **note's panel**

### ISSUE THREAD

**Issue by EdoFro** [freeplane/freeplane#608]
## **Is your feature request related to a problem? Please describe.**
Using Tool Panel and Note Panel at the same time uses too much screen space, so when I use the Note panel I always hide the Tool panel.
Sometimes I want to change the content type of the note and I need to unhide the Tool panel only for that.

I would like to have the posibility to change the "Content type" directly when editing a note in the Note Panel (with the mouse and keyboard only too)

## **Describe the solution you'd like**
I would like to have a dropbox or a menu command or a button or a keyboard shortcut that makes it easy to change content type for a node's note when (or before) editing it in the Note panel.

I imagined something like this:
![image](https://user-images.githubusercontent.com/35700575/187548179-8a9177bf-13e8-493c-b479-e387d037ffbb.png)

## **Describe alternatives you've considered**
Adding 3 content type dropboxes for node's text, details and note into Freeplane toolbar.

Thanks in advance,
edo

**Comment by euu2021** [comment]
in the API, I see the method setNoteContentType. Perhaps, a solution is a script that creates a GUI with a combo box (or radio button) to pick up the content type.

**Comment by EdoFro** [comment]
> in the API, I see the method setNoteContentType. Perhaps, a solution is a script that creates a GUI with a combo box (or radio button) to pick up the content type.

Yes, I made a script for myself, but I still think it could be a good UI improvement.

```groovy
import groovy.swing.SwingBuilder
import javax.swing.*

swingBuilder = new SwingBuilder()

def n = node

def titulo = 'Change note\'s content type'
def mensaje = "current: '${n.noteContentType}'"
def opciones = ['auto','html','latex','markdown']

n.noteContentType = respuestaDialogo(opciones,mensaje,titulo)

def respuestaDialogo(options,msg,title){
    def pane = swingBuilder.optionPane(message:msg, selectionValues:options, optionType:JOptionPane.CLOSED_OPTION)
    pane.createDialog(ui.frame, title).show()
    return pane.inputValue
}
```
